packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -332,6 +340,40 @@ - (bool)loadUrl:(NSString*)url withHeaders:(NSDictionary<NSString*, NSString*>*)
   return true;
 }
 
+- (NSMutableString*)createPostParams:(NSDictionary<NSString*, NSObject*>*)params {
+  NSMutableString* resultString = [NSMutableString string];
+  for (NSString* key in [params allKeys]) {
+    if ([resultString length] > 0) [resultString appendString:@"&"];
+    [resultString appendFormat:@"%@=%@", key, [params objectForKey:key]];
+  }
+  return resultString;
+}
+
+- (bool)loadUrlPost:(NSString*)url
+        withHeaders:(NSDictionary<NSString*, NSString*>*)headers
+      initialParams:(NSDictionary<NSString*, NSObject*>*)params {
+  NSURL* nsUrl = [NSURL URLWithString:url];
+  if (!nsUrl) {
+    return false;
+  }
+
+  NSMutableString* resultString = [self createPostParams:params];
+
+  NSMutableURLRequest* request = [NSMutableURLRequest requestWithURL:nsUrl];
+
+  NSData* postData = [resultString dataUsingEncoding:NSUTF8StringEncoding allowLossyConversion:YES];
+  NSString* contentLength = [NSString stringWithFormat:@"%ld", (unsigned long)postData.length];
+
+  [request setHTTPMethod:@"POST"];
+  [request setHTTPBody:postData];
+  [request setValue:contentLength forHTTPHeaderField:@"Content-Length"];
+  [request setValue:@"application/x-www-form-urlencoded" forHTTPHeaderField:@"Content-Type"];
```

Comment:
Seems like the Android webview forces this encoding, do you know if `multipart/form-data` is allowed with WkWebView?

Reply:
I don't know if you can send multipart like that, but code I've seen uses form encoded only